ydb/core/blobstorage/vdisk/defrag/defrag_search.h

```diff
@@ -337,25 +338,20 @@ namespace NKikimr {
         bool Scan(TDuration quota, THullDsSnap fullSnap) {
             // create iterator and set it up to point to next blob of interest
             TLogoBlobsSnapshot::TForwardIterator iter(fullSnap.HullCtx, &fullSnap.LogoBlobsSnap);
-            if (NextId) {
-                iter.Seek(*NextId);
-            } else {
-                iter.SeekToFirst();
-            }
-
+            THeapIterator<TKeyLogoBlob, TMemRecLogoBlob, true> heapIt;
```

Comment:
it would be nice to have constructor allowing to write this as
... heapIt(&iter);
showing:
1. We keep reference to iter, so we need it.
2. We construct HeapIterator over existing one.
(instead of PutToHeap construct)